packages/client/test/client.spec.ts

```diff
@@ -42,10 +42,12 @@ describe('test utils', () => {
     localStorage.clear();
   });
   it('test LocalStorage Mock', () => {
+    expect(getData(DataType.Key, StorageType.LocalStorage).orUndefined()).to.be.equal(undefined);
+    expect(getData(DataType.Message, StorageType.LocalStorage).orUndefined()).to.be.equal(undefined);
```

Comment:
Because of the return value wrapped as an optional type, optional wrapping must be unwrapped in some tests.